tools/run-integration-tests.py

```diff
@@ -25,12 +25,15 @@
 PLATFORM_SERIES_TESTS = {
     "azuregeneric": ["xenial", "bionic", "focal"],
     "azurepro": ["xenial", "bionic", "focal"],
+    "azurepro-fips": ["xenial", "bionic", "focal"],
     "awsgeneric": ["xenial", "bionic", "focal"],
     "awspro": ["xenial", "bionic", "focal"],
+    "awspro-fips": ["xenial", "bionic", "focal"],
```

Comment:
Since I am working on the gcp pro fips integration tests, should I also add this entry for it as well, when am doing the PR?

Reply:
Oh yes, if you could please add the line here (: